packages/api-reference/src/components/Badge/Badge.vue

```diff
@@ -15,4 +23,19 @@
   display: inline-block;
   text-transform: uppercase;
 }
+
+.green {
+  color: var(--scalar-color-1);
+  background: var(--scalar-color-green);
+}
```

Comment:
what do you think about dimming down background on dark mode to maintain color accessibility like:

```suggestion
.dark-mode .green {
  color: var(--scalar-color-green);
  background: color-mix(in srgb, var(--scalar-color-green) 15%, transparent);
}
```

Reply:
i'll let @cameronrohani jump in - great work anyway tysm! @WDaan